### PR TITLE
Create bangs.json

### PR DESCRIPTION
**Pull request by tEctutoRIt**
Simple regex to create "bang" for different services. A concept introduced by DDG (https://duckduckgo.com/bangs) It can be very useful for example if you get a text you about a movie just share the name to URLCheck and add !i for IMDb Search It is simple and offline.

### CODE REVIEW

**Review by salazarbarrera — COMMENTED** [review]
The 9th line:
`    "replacement": "https:\/\/morty.ononoki.org\/?mortyurl=https%3A%2F%2Fyts.mx%2Fbrowse-movies%2F$1",`

why does it have a reference to MortyProxy instead of directly taking you to YTS like the other links?

**Comment by tEctutoRIt** [comment]
> why does it have a reference to MortyProxy instead of directly taking you to YTS like the other links?

It's because yts is blocked in my country so I use it as proxy + it removes unnecessary elements. Want me to change it?

**Comment by salazarbarrera** [comment]
No, I don't think that's necessary. Changing the name of the filter to something like "Proxyfied YTS  search" will suffice.

**Comment by tEctutoRIt** [comment]
Ok done

**Review by salazarbarrera — COMMENTED** [review]
Yeah, I think that will do

**Review by salazarbarrera — APPROVED** [review]

**Review by salazarbarrera — COMMENTED** [review]
Sadly, I haven't writing privileges, I thought I had after the pull request I got accepted weeks ago, but it was not the case 😞. sorry

**Comment by Raj9039852537** [comment]
With proxified yts also add non proxified one for other peoples and I will merge the request

File: bangs.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "Where to Watch": {
+    "regex": "^(.*)!n$",
+    "replacement": "https:\/\/www.justwatch.com\/in\/search?q=$1",
+    "enabled": "true"
+  },
+  "Proxyfied YTS Search": {
```
Comment (Raj9039852537) [inline bangs.json:7]: "YTS Search": {
    "regex": "^(.*)!y$",
    "replacement": "https:\/\/yts.mx\/browse-movies\/$1",
    "enabled": "true"
  },

Comment (tEctutoRIt) [reply bangs.json:7]: Okay

**Review by Raj9039852537 — APPROVED** [review]

**Pull request merged**